core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -2933,17 +2933,21 @@ components:
     LtsTransactionConstructionResponse:
       type: object
       required:
-        - epoch
+        - current_epoch
+        - ledger_clock
       properties:
-        epoch:
+        current_epoch:
           type: integer
           format: int64
           minimum: 0
           maximum: 10000000000
           description: An integer between `0` and `10^10`, marking the current epoch
-###################################
+        ledger_clock:
+          $ref: '#/components/schemas/Instant'
+          description: The latest round proposer's timestamp (rounded down to the current minute).
```

Comment:
The reason for including this field is to detect whether the node is synced up - so it's probably worth mentioning in the description that clients may wish to double-check this field before blindly using `current_epoch` for construction.

Alternatively, we could allow an (optional) request parameter with "acceptable_delay_seconds" which causes a 400 to be returned if the node isn't sufficiently synced up

Reply:
Unless we can offer a more accurate but complex logic for node status I would leave this (simple) behaviour to the client.